### DISCUSSION TITLE
Error using the polar wake OpenFAST v3.5.0 FAST Farm

### DISCUSSION THREAD

**Discussion by Chen1995SYSU** [OpenFAST/openfast#1737] [Q&A]
Dear Jason，
I received this error，but I don't know how to solve it.
I have checked all the files, but I don't know how to change them. Can you give me some suggestions?
<img width="855" alt="f6d3f994672a0e376b3146ca5edc909" src="https://github.com/OpenFAST/openfast/assets/142133415/56fc5f72-781d-485a-9e2b-66037066b9f3">

<img width="417" alt="b47b1be7146166099bb742b2e071ba4" src="https://github.com/OpenFAST/openfast/assets/142133415/c3fdec4d-6212-4efa-9ac7-08ec6be2a5b1">

**Comment by Chen1995SYSU** [comment]
I reduce the Dt_Low, but can't solve the issue

**Reply by jjonkman** [reply]
Dear @Chen1995SYSU,

It looks like you are receiving a series of warnings about the BEM solution for the two downstream wind turbines well before the error regarding the propeller-brake region is triggered.  Can you clarify what wind conditions you are simulating via InflowWind?  Have you initialized the rotor speed and blade-pitch angle of each wind turbine properly (conditioned on the mean hub-height wind speed you are simulating)?  What does the wind speed into the rotor, rotor speed, and blade pitch angle of each wind turbine look like over the simulation?

Best regards,
